concepts/packages/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "blurb": "TODO: add blurb",
   "authors": ["ErikSchierboom"],
-  "contributors": []
+  "contributors": ["bobtfish"]
```

Comment:
Personal opinion, your username belongs in the authors line since you added so much material here for the concept. In contrast, for the exercise this did not change much, but for the concept documentation it added over half of the material.